.grype.yml

```diff
@@ -0,0 +1,49 @@
+# List of vulnerabilities to ignore for the anchore scan
+# https://github.com/anchore/grype#specifying-matches-to-ignore
+# More info can be found in the docs/infra/vulnerability-management.md file
+
+# List of vulnerabilities to ignore for the anchore scan
+# https://github.com/anchore/grype#specifying-matches-to-ignore
+# More info can be found in the docs/infra/vulnerability-management.md file
```

Comment:
Why is the comments there twice?

Reply:
not sure, maybe something from my copy paste. thanks for catching that

Reply:
oh it's because the template config file has it duplicated.

Reply:
https://github.com/navapbc/template-infra/blob/main/.grype.yml#L1-L7 

haha, must have been your own copy pasta :P i must not have caught it when reviewing #203